src/RequestClientWrapper.ts

```diff
@@ -366,8 +366,14 @@ export default class RequestClientWrapper {
   private parseJwt(token: string) {
     let output: string;
     try {
-      return JSON.parse(Buffer.from(token.split('.')[1], 'base64').toString());
+      const base64Url = token.split('.')[1];
+      const base64 = base64Url.replace(/-/g, '+').replace(/_/g, '/');
+      const jsonPayload = decodeURIComponent(atob(base64).split('').map((c) =>  {
+        return '%' + ('00' + c.charCodeAt(0).toString(16)).slice(-2);
+      }).join(''));
+      return JSON.parse(jsonPayload);
     } catch (e) {
+      console.error(`Error parsing JWT token: ${e}`);
```

Comment:
@cduda I would argue against logging an error here, as this will always show an error log when the non-embedded angular webapp uses it.  If anything, maybe an info log stating something like: `Session token found` or `JWT token found`.